index.vue

```diff
@@ -93,7 +98,7 @@ export default
 			when @isDetached and @revealed then 0
 
 			# Else, hide
-			else -1 * @height
+			else (-1 * @height) - @offScreenPad
```

Comment:
With the other change I'm recommending, this just becomes:

```suggestion
			else -1 * @offscreenHeight
```